app/src/main/res/layout/item_tab_details.xml

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layout xmlns:android="http://schemas.android.com/apk/res/android">
```

Comment:
should remove layout tag as you don't need to use data tag

Reply:
i used it to use databinding for type and null safety

Reply:
👍